server/server_utils/requests.c

```diff
@@ -0,0 +1,43 @@
+#include "../inc/server.h"
+
+// Get the type of the request from the read json object
+t_request_type get_request_type(cJSON* json) {
```

Comment:
Make it static and remove from header

Reply:
it will be used in some other funcs in future so it should stay as it is